diff --git a/ks_includes/widgets/scroll.py b/ks_includes/widgets/scroll.py
index e9accaa..f698f79 100644
--- a/ks_includes/widgets/scroll.py
+++ b/ks_includes/widgets/scroll.py
@@ -6,7 +6,7 @@ from gi.repository import Gdk, Gtk
 
 class CustomScrolledWindow(Gtk.ScrolledWindow):
     def __init__(self, steppers=False, **kwargs):
-        args = {"hexpand": True, "vexpand": True, "overlay_scrolling": False}
+        args = {"hexpand": True, "vexpand": True, "overlay_scrolling": True}
         args.update(kwargs)
         super().__init__(**args)
         self.add_events(
